src/declarations/stencil-public-runtime.ts

```diff
@@ -596,12 +596,12 @@ export interface ChildNode {
  *
  * For further information: https://stenciljs.com/docs/host-element
  */
-export declare const Host: FunctionalComponent<HostAttributes>;
+export declare const Host: (props: HostAttributes) => VNode;
 
 /**
  * Fragment
  */
-export declare const Fragment: FunctionalComponent<{}>;
```

Comment:
Similar to my comment above - while I understand the motivation for this change, it would also be a breaking change.

In Stencil v4, the following code type checks without error:
```ts
import { Fragment } from '@stencil/core';

const Hello: typeof Fragment = (_props: any, children: any, utils: any) =>
  utils.map(children, child => ({
    ...child,
    vattrs: {
      ...child.vattrs,
      class: `${child.vattrs.class} add-class`,
    },
  }));
```

Whereas with this commit/change, it fails with:
```
[ ERROR ]  TypeScript: src/components/my-component/my-component.tsx:4:7
           Type '(_props: any, children: any, utils: any) => any' is not assignable to type '(props: {}) =>
           VNode'.Target signature provides too few arguments. Expected 3 or more, but got 1.

      L4:  const Hello: typeof Fragment = (_props: any, children: any, utils: any) =>
      L5:    utils.map(children, child => ({
```